src/ematlab.ts

```diff
@@ -7,6 +7,10 @@ import * as core from "@actions/core";
 
 export const rootFile = path.join(os.tmpdir(), "ephemeral_matlab_root");
 
+interface ProcessEnv {
+    [key: string]: string | undefined
+}
```

Comment:
I think this is already defined in TypeScript as:

`interface ProcessEnv extends Dict<string> {}` 

Could just use that?

Reply:
Ah, I tried it before, and it wasn't finding it during test execution. Looks like it needed to be referenced as `NodeJs.ProcessEnv`.